src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReferralHelper.java

```diff
@@ -0,0 +1,352 @@
+import uk.gov.hmcts.et.common.model.ccd.types.ReferralType;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.UserService;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Component
+public class ReferralHelper {
+    private final UserService userService;
+    private static final String TRUE = "True";
+    private static final String FALSE = "False";
+    private static final String JUDGE_ROLE_ENG = "caseworker-employment-etjudge-englandwales";
+    private static final String JUDGE_ROLE_SCOT = "caseworker-employment-etjudge-scotland";
+    private static final String GUIDANCE_DOC_LINK = "<hr>To help you complete this form, open the "
+        + "<a href=\"url\">referral guidance documents</a>";
+    private static final String HEARING_DETAILS = "<hr><h3>Hearing details %s</h3>"
+        + "<pre>Date &nbsp;&#09&#09&#09&#09&#09&nbsp; %s"
+        + "<br><br>Hearing &#09&#09&#09&#09&nbsp; %s"
+        + "<br><br>Type &nbsp;&nbsp;&#09&#09&#09&#09&#09 %s</pre>";
+
+    private static final String REFERRAL_DETAILS = "<h3>Referral</h3>"
+        + "<pre>Referred by &nbsp;&#09&#09&#09&#09&#09&#09&#09&#09&#09&nbsp; %s"
+        + "<br><br>Referred to &nbsp;&nbsp;&#09&#09&#09&#09&#09&#09&#09&#09&#09&nbsp; %s"
+        + "<br><br>Email address &nbsp;&#09&#09&#09&#09&#09&#09&#09&#09&nbsp; %s"
+        + "<br><br>Urgent &nbsp;&#09&#09&#09&#09&#09&#09&#09&#09&#09&#09&#09&nbsp; %s"
+        + "<br><br>Referral date &#09&#09&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Next hearing date &#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Referral subject &#09&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Details of the referral &#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Documents &nbsp;&nbsp;&#09&#09&#09&#09&#09&#09&#09&#09&#09&nbsp; %s"
+        + "<br><br>Recommended instructions &nbsp;&#09&#09&#09&nbsp; %s</pre><hr>";
+
+    private static final String REPLY_DETAILS = "<h3>Reply %s</h3>"
+        + "<pre>Reply by &nbsp;&nbsp;&#09&#09&#09&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Reply to &nbsp;&nbsp;&#09&#09&#09&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Email address &nbsp;&#09&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Urgent &nbsp;&#09&#09&#09&#09&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Referral date &nbsp;&nbsp;&#09&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Hearing date &nbsp;&nbsp;&#09&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Referral subject &nbsp;&nbsp;&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Directions &nbsp;&nbsp;&nbsp;&#09&#09&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>Documents &nbsp;&#09&#09&#09&#09&#09&#09&#09&#09&#09 %s"
+        + "<br><br>General notes &nbsp;&#09&#09&#09&#09&#09&#09&#09&#09 %s</pre><hr>";

```

Comment:
Can we omit the "General notes" and "Documents" if nothing is provided for it? Could look into using a StringBuilder instead to make omitting things easier

<img width="515" alt="image" src="https://user-images.githubusercontent.com/43950645/186627982-4b50746d-f35e-415c-a3f8-70a7ff1d8e49.png">